recipes/rattler-build/conda_build_config.yaml

```diff
@@ -0,0 +1,3 @@
+# We need fmemopen in libsolv which became available in 10.13
+MACOSX_SDK_VERSION:        # [osx and x86_64]
```

Comment:
You need `MACOSX_DEPLOYMENT_TARGET` not the SDK unless you are using an availability check.

Reply:
Thanks!